synthtool/gcp/templates/node_library/README.md

```diff
@@ -0,0 +1,117 @@
+[//]: # "This README.md file is auto-generated, all changes to this file will be lost."
+[//]: # "To regenerate it, use `python -m synthtool`."
+<img src="https://avatars2.githubusercontent.com/u/2810941?v=3&s=96" alt="Google Cloud Platform logo" title="Google Cloud Platform" align="right" height="96" width="96"/>
+
+# [{{ metadata['repo']['name_pretty'] }}: {{ metadata['repo']['language']|language_pretty }} Client](https://github.com/{{ metadata['repo']['repo'] }})
+
+{{ metadata['repo']['release_level']|release_quality_badge }}
+[![npm version](https://img.shields.io/npm/v/{{ metadata['name'] }}.svg)](https://www.npmjs.org/package/{{ metadata['name'] }})
+[![codecov](https://img.shields.io/codecov/c/github/{{ metadata['repo']['repo'] }}/master.svg?style=flat)](https://codecov.io/gh/{{ metadata['repo']['repo'] }})
+
+{{ metadata['description'] }}
+{% if metadata['deprecated'] %}
+| :warning: Deprecated Module |
+| --- |
+| This library is **deprecated**. {{ metadata['deprecated'] }} |
+{% endif %}
+* [Using the client library](#using-the-client-library){% if metadata['samples']|length %}
+* [Samples](#samples){% endif %}
+* [Versioning](#versioning)
+* [Contributing](#contributing)
+* [License](#license)
+
+## Using the client library
+
+1.  [Select or create a Cloud Platform project][projects].
+1.  [Enable billing for your project][billing].
```

Comment:
we used to be able to hide this step for libraries that don't require billing; do we still need this @JustinBeckwith, how common is this?

Reply:
I did some sleuthing about, and Firestore is the only one I see using it:
https://github.com/googleapis/nodejs-firestore/blob/master/.cloud-repo-tools.json#L6

That having been said - there are plenty of our products that have a free tier:
https://cloud.google.com/free/

My 2 cents: I would like to see us keep this field, and actually set it correctly, because I think requiring billing is a huge leap for a lot of developers that just want to play on the platform.

Reply:
(also sorry for creating work)

Reply:
[Quickstarts](https://cloud.google.com/vision/docs/quickstart-client-libraries) on cloud.google.com seem to always have the billing step (even products listed at cloud.google.com/free. Maybe our READMEs should also always have this step, for consistency?

As an aside, it kind of makes sense that the billing step was suppressed for firestore, since their [Get Started](https://firebase.google.com/docs/firestore/quickstart) doesn't explicitly mention it. :thinking: